src/endpoints/endpoints.service.ts

```diff
@@ -121,7 +149,23 @@ export class EndpointsService {
         .where('id = :endpointId', { endpointId })
         .returning('*')
         .execute();
-
+      const newValues = await this.endpointRepo
+        .createQueryBuilder()
+        .select(values)
+        .where('id = :enpointId', {endpointId})
+        .execute()
+      //LOG THE UPDATE MADE
+      const logger = await this.loggerRepo.create({
+        entity_type: 'ENDPOINT',
+        identifier: endpointId,
+        action_type: Action.Update,
+        previous_values: previousValues,
+        new_values: newValues,
+        operated_by: email
+      })
+      await this.loggerRepo.save(logger)
+      console.log(logger);
```

Comment:
I think it will be better to run this synchronously, to avoid longer response time. verify from brain. Also cleanup the console logs

Reply:
ur having migration error?
... wow i did not have any of those sha